Adyen/Assets/en-US.lproj/Localizable.strings

```diff
@@ -21,6 +21,7 @@
 "adyen.card.numberItem.placeholder" = "1234 5678 9012 3456";
 "adyen.card.numberItem.invalid" = "Invalid card number";
 "adyen.card.expiryItem.title" = "Expiry date";
+"adyen.card.expiryItem.title.optional" = "Expiry date (optional)";
```

Comment:
We already have line for `Expiry date` and line for `(optional)` and we previously used combination of them.
why change this?
Also you need to request translation for "CVC/CVV (optional)"

Reply:
We already had CVC/CVV (optional) translation in and it is used in FormCardSecurityCodeItem. That's why I did the same. 
But also, some languages may differ when writing texts together. The word optional may be different when next to ExpDate etc.

Reply:
ok, you right.